tracing-subscriber/src/filter/subscriber_filters/mod.rs

```diff
@@ -475,7 +475,7 @@ macro_rules! filter_impl_body {
         }
 
         #[inline]
-        fn max_level_hint(&self) -> Option<LevelFilter> {
+        fn max_level_hint(&self) -> ::core::option::Option<LevelFilter> {
```

Comment:
this macro is only used internally; it's not publicly re-exported. i think it would be fine to continue using the prelude, or add a module-level import here?